server/polar/customer_portal/endpoints/subscription.py

```diff
@@ -14,25 +15,27 @@
 from polar.postgres import get_db_session
 from polar.product.schemas import ProductID
 from polar.routing import APIRouter
+from polar.subscription.service import AlreadyCanceledSubscription
 
 from .. import auth
 from ..schemas.subscription import (
     CustomerSubscription,
     CustomerSubscriptionUpdate,
 )
 from ..service.subscription import (
-    AlreadyCanceledSubscription,
     CustomerSubscriptionSortProperty,
 )
 from ..service.subscription import customer_subscription as user_subscription_service
 
+log = structlog.get_logger()
+
 router = APIRouter(
     prefix="/subscriptions", tags=["subscriptions", APITag.documented, APITag.featured]
 )
 
-SubscriptionID = Annotated[UUID4, Path(description="The subscription ID.")]
+SubscriptionID = Annotated[UUID4, Path(description="Your subscription ID.")]
 SubscriptionNotFound = {
-    "description": "Subscription not found.",
+    "description": "Your subscription was not found.",
```

Comment:
Not sure about the "Your..." style. This is not something shown to the end-user, only in the API docs.

Reply:
Fair and I agree. I wanted to distinguish it more as a few developers have historically been confused between core API endpoints vs. `/customer-portal/*`, but can achieve the same with more clarity vs. `your`. Will fix.